test/integration/suites/batchx509SVID/README.md

```diff
@@ -0,0 +1,17 @@
+# Batchx509SVID RPC Integration Test
+
+## Overview
+
+This test ensures the continued operation of the `Batchx509SVID` RPC in Open Source SPIRE.
+
+## Test Steps
+
+1. **Setup**
+
+    - Starts SPIRE server and agent.
+    - Configures necessary registration entries.
+
+   Run the setup script:
+
+   ```bash
+   sudo ./test/integration/suites/batchx509svid/01-setup.sh
```

Comment:
We should not directly execute this shell. Rather we should hook into the framework such that this can be invoked via `test-one.sh` such that it properly runs without special privileges as part of the `make integration` target.